lms/djangoapps/certificates/admin.py

```diff
@@ -28,6 +29,11 @@ def __init__(self, *args, **kwargs):
         self.fields['organization_id'] = forms.TypedChoiceField(
             choices=org_choices, required=False, coerce=int, empty_value=None
         )
+        lang_choices = list(settings.CERTIFICATE_TEMPLATE_LANGUAGES)
+        lang_choices.insert(0, (None, '------------'))
```

Comment:
It might be clearer to list this as 'All Languages', rather than '------------', since the effect of selecting this will be that the template is used for courses in any language.